.github/workflows/main.yml

```diff
@@ -403,9 +403,13 @@ jobs:
       uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.3
       with:
         name: unbundled_m2repo
-        path: |
-          unbundled-aaos/out/aaos-apps-gradle-build/unbundled_m2repo/
-          unbundled-aaos/prebuilts/sdk/34/
+        path: unbundled-aaos/out/aaos-apps-gradle-build/unbundled_m2repo/
+
+    - name: Upload aaos prebuilts
```

Comment:
I created a separate task and then it fixed the error. :D

Reply:
Good job :)